src/test/java/com/mael/payroll/transactions/AddHourlyEmployeeTest.java

```diff
@@ -0,0 +1,45 @@
+package com.mael.payroll.transactions;
+
+import com.mael.payroll.*;
+import com.mael.payroll.paymentSchedules.HourlySchedule;
+import com.mael.payroll.paymentTypes.HourlyPayment;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class AddHourlyEmployeeTest {
+
+    private Employee employee;
+
+    @Before
+    public void setUp() {
+        PayrollDBFacade payrollDB         = new PayrollDBFacade();
+        AddEmployeeTransaction addHourlyE = new AddHourlyEmployee(1, "Squiddo", "FishBowl", 1000.0);
```

Comment:
Same as above. Are the values `"FishBowl"` and `1000.0` needed at all? If so, they should have matching test.